examples/tokenvm/controller/action_indexer.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package controller
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/examples/tokenvm/actions"
+	"github.com/ava-labs/hypersdk/extension/indexer"
+)
+
+var _ indexer.SuccessfulTxSubscriber = (*actionHandler)(nil)
+
+type actionHandler struct {
```

Comment:
Similar comment as the corresponding comment in morpheusvm

Reply:
Renaming

Reply:
Done